drivers/clock_control/Kconfig.fixed

```diff
@@ -0,0 +1,10 @@
+# Fixed clock control driver config
+
+# Copyright (c) 2022 Google, LLC
+# SPDX-License-Identifier: Apache-2.0
+
+config CLOCK_CONTROL_FIXED_CLOCK
+	bool "Fixed Clock Clock Control"
+	default n
```

Comment:
I think the default should come from whether any fixed clock instances are enabled in the devicetree (see https://github.com/zephyrproject-rtos/zephyr/blob/8cb69f2c6b2e49047b46b79a446b6f65e2f9f9d5/drivers/clock_control/Kconfig.stm32#L39).

Reply:
I agree, this needs to be opt-in. Many platforms won't use this at all, since it introduces overhead for little/no gain with the current clock control "API". However, it should at least depend on `DT_HAS_FIXED_CLOCK_ENABLED`.

Reply:
Where is the DT_HAS_FIXED_CLOCK_ENABLED defined or what defines it?

Reply:
It is generated from parsing devicetree.

Reply:
Done.